src/envo/envo-edit.owl

```diff
@@ -21961,7 +22154,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000275> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_01000276> (ecoregion)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "DOI:10.1641/0006-3568(2001)051[0933:TEOTWA]2.0.CO;2") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "URL:http://worldwildlife.org/biomes") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000276> "A large unit of land or water containing a geographically distinct assemblage of species, natural communities, and environmental conditions.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://worldwildlife.org/biomes>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://dx.doi.org/10.1641/0006-3568(2001)051[0933:TEOTWA]2.0.CO;2>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000276> "A large unit of land or water containing a geographically distinct assemblage of species, natural communities, and environmental conditions.")
```

Comment:
This is invalid RDF/XML. the OWLAPI lets it by, but more struct RDF/XML parsers, such the Rust parsers we increasingly rely on will choke on this, it needs to be URL encoded

(note that the syntax was fine when it was a string literal)

Reply:
https://dx.doi.org/10.1641/0006-3568%282001%29051%5B0933:TEOTWA%5D2.0.CO;2